### PR TITLE
SLIDESNET-42342 - Add article for SLIDESNET-36994

### PR DESCRIPTION
**Pull request by MaxChuryumov**
https://issue.lutsk.dynabic.com/issues/SLIDESNET-42342

### CODE REVIEW

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+title: Custom Shape
+type: docs
+weight: 10
+url: /net/custom-shape/
+---
+
+# Customization of the Shape Geometry (Shape Points Editing)
+  * [Add Custom Points to the Shape](#add-custom-points-to-the-shape-)
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:9]: The navigation link is not working
Please change  #add-custom-points-to-the-shape- to #add-custom-points-to-the-shape

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+---
+title: Custom Shape
+type: docs
+weight: 10
+url: /net/custom-shape/
+---
+
+# Customization of the Shape Geometry (Shape Points Editing)
+  * [Add Custom Points to the Shape](#add-custom-points-to-the-shape-)
+  * [Remove Points From the Shape](#remove-points-from-the-shape-)
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:10]: The navigation link is not working

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+weight: 10
+url: /net/custom-shape/
+---
+
+# Customization of the Shape Geometry (Shape Points Editing)
+  * [Add Custom Points to the Shape](#add-custom-points-to-the-shape-)
+  * [Remove Points From the Shape](#remove-points-from-the-shape-)
+  * [Create Custom Shape](#create-custom-shape)
+  * [Create Composite Custom Shape](#create-composite-custom-shape)
+  * [Conversion of GeometryPath to GrpahicsPath (System.Drawing.Drawing2D)](#-onversion-of-geometrypath-to-grpahicspath--systemdrawingdrawing2d-)
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:13]: The navigation link is not working 
typo #-onversion, it should be #conversion

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  
+## Add Custom Points to the Shape
+- Create an instance of the [GeometryShape](https://apireference.aspose.com/slides/net/aspose.slides/geometryshape) class of type [ShapeType.Rectangle](https://apireference.aspose.com/slides/net/aspose.slides/shapetype)
+- Retrieve an instance of the [GeometryPath]() class from the shape
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:17]: empty link [GeometryPath](), is it correct?

**Review by stasmiroshnichenko — COMMENTED** [review]
Please see my comments.
You can check your changes directly in GitHub https://github.com/aspose-slides/Aspose.Slides-Documentation/blob/SLIDESNET-42342/net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md

**Review by stasmiroshnichenko — APPROVED** [review]

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+```
+**Adds cubic Bezier curve** at the end the path:
+``` csharp
+void CubicBezierTo(PointF point1, PointF point2, PointF point3);
+void CubicBezierTo(float x1, float y1, float x2, float y2, float x3, float y3);
+```
+**Adds cubic Bezier curve** to the specified place of the path:
+``` csharp
+void CubicBezierTo(PointF point1, PointF point2, PointF point3, uint index);
+   void CubicBezierTo(float x1, float y1, float x2, float y2, float x3, float y3, uint index);
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:43]: extra tab

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+void QuadraticBezierTo(PointF point1, PointF point2, uint index);
+void QuadraticBezierTo(float x1, float y1, float x2, float y2, uint index);
+```
+**Appends the specified arc** to the path:
+``` csharp
+    void ArcTo(float width, float heigth, float startAngle, float sweepAngle);
+```
+**Closes the current figure** of this path:
+``` csharp
+    void CloseFigure();
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:61]: extra tab

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+void QuadraticBezierTo(float x1, float y1, float x2, float y2);
+```
+**Adds quadratic Bezier curve** to the specified place of the path:
+``` csharp
+void QuadraticBezierTo(PointF point1, PointF point2, uint index);
+void QuadraticBezierTo(float x1, float y1, float x2, float y2, uint index);
+```
+**Appends the specified arc** to the path:
+``` csharp
+    void ArcTo(float width, float heigth, float startAngle, float sweepAngle);
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:57]: extra tab

File: net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+IGeometryPath provides methods for adding segments of various types:
+
+**Adds line** to the end of the path
+``` csharp
+void LineTo(PointF point);
+void LineTo(float x, float y);
+```
+**Adds line** to the specified place of the path:
+``` csharp    
+void LineTo(PointF point, uint index)
```
Comment (stasmiroshnichenko) [inline net/developer-guide/presentation-content/powerpoint-shapes/shape-types/custom-shapes/_index.md:32]: a semicolon is missed at the end of the line

**Review by stasmiroshnichenko — CHANGES_REQUESTED** [review]

**Review by stasmiroshnichenko — APPROVED** [review]

**Review by yury-knml — APPROVED** [review]

**Review by agrazhevskaja — CHANGES_REQUESTED** [review]
@MaxChuryumov Please, review my comments under your commits and fix them. Thank you

**Review by agrazhevskaja — APPROVED** [review]

**Pull request merged**